languages.toml

```diff
@@ -3680,6 +3680,19 @@ language-servers = [
 indent = { tab-width = 2, unit = "  " }
 grammar = "javascript"
 
+[[language]]
+name = "gherkin"
+scope = "source.feature"
+file-types = ["feature"]
+comment-token = "#"
+indent = { tab-width = 2, unit = "  " }
+grammar = "gherkin"
+roots = []
```

Comment:
```suggestion
```

`roots` is defaulted to `[]` so we can remove this line

Reply:
Removed